### PR TITLE
Mbs 8007 sentry link

### PR DESCRIPTION
**Pull request by dsvoronin**

### CODE REVIEW

**Review by int02h — APPROVED** [review]

File: subprojects/gradle/sentry-config/src/main/kotlin/com/avito/android/sentry/ProjectExtensions.kt [hunk 1]
```diff
@@ -7,47 +7,66 @@ import com.avito.kotlin.dsl.ProjectProperty
...
+        if (!projectUrl.isNullOrBlank() && !buildId.isNullOrBlank()) {
+            project.gradle.buildFinished {
+
+                val url = HttpUrl.parse(projectUrl)
+                    ?.newBuilder()
+                    ?.addQueryParameter("query", "$buildIdTag:$buildId")
+                    ?.build()
+                    ?.toString()
+
+                project.logger.lifecycle("Build errors: $url")
```
Comment (eugene-krivobokov) [inline subprojects/gradle/sentry-config/src/main/kotlin/com/avito/android/sentry/ProjectExtensions.kt:65]: This url is incorrect in local builds: `build_id:0`

I see two options:
- Surrogate build id for local builds
- Filter by available attributes: host, user, ...

What do you think?

File: subprojects/gradle/sentry-config/src/main/kotlin/com/avito/android/sentry/ProjectExtensions.kt [hunk 2]
```diff
@@ -7,47 +7,66 @@ import com.avito.kotlin.dsl.ProjectProperty
...
             environment = info.environment.publicName,
             serverName = info.node ?: "unknown",
             release = info.commit ?: "unknown",
             tags = tags
         )
+
+        val projectUrl = project.getOptionalStringProperty("avito.sentry.projectUrl")
+
+        if (!projectUrl.isNullOrBlank() && !buildId.isNullOrBlank()) {
+            project.gradle.buildFinished {
```
Comment (eugene-krivobokov) [inline subprojects/gradle/sentry-config/src/main/kotlin/com/avito/android/sentry/ProjectExtensions.kt:57]: I think this must be the responsibility of another abstraction.

File: subprojects/gradle/sentry-config/src/main/kotlin/com/avito/android/sentry/ProjectExtensions.kt (hunk 1 above)
Comment (sergioboi) [reply subprojects/gradle/sentry-config/src/main/kotlin/com/avito/android/sentry/ProjectExtensions.kt:65]: Lets create local id from user-machine-and-date-time

Comment (eugene-krivobokov) [reply subprojects/gradle/sentry-config/src/main/kotlin/com/avito/android/sentry/ProjectExtensions.kt:65]: MBS-8612

**Pull request merged**